static/oas/v1/rosters/rosters.yml

```diff
@@ -0,0 +1,84 @@
+paths:
+  rosters:
+    post: 
+      tags: 
+        - roster
+      summary: Create Roster  
+      requestBody:
+        description: A JSON object with a valid roster
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/schemas/PostRosterRequest"  
+      responses: 
+        200:
+          description: Successfully create a roster by captain
+          content: 
+            application/json: 
+              schema: 
+                $ref: '#/schemas/PostRosterResponse'
```

Comment:
I believe everywhere where you see #/schemas... you need to change it to #/components/schemas...
This is what I am seeing in the current code.

Reply:
changed